src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
+        };
```

Comment:
Consider using abstract references for the array elements instead of specific ones to allow for polymorphism in your code. For example: `Machine truck = new Truck();`